script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -87,15 +87,15 @@ It is an Autumn morning in October, a time when all traces of summer have disapp
 <Page24>
 Thinking about it, I've been gazing at this scenery for almost a year and a half.
 This is the last time I will see this scenery on my train ride to school.
-I think back on the past seven years, tracing the landscape as I pass by.
+I rethink of the past seven years, tracing the landscape as I pass by.
 <Page25>
 When I was ten years old, I recovered from a serious accident that should have killed me instantly.
-I met Sensei.
+I have met Sensei.
```

Comment:
I think this changes the meaning a bit - "I met Sensei" reads as "meeting sensei is an event that happened in the past", but "I have met Sensei" is saying "I exist in a state of having met Sense", which I don't think is the intent here